src/legacy/core_plugins/kibana/public/management/sections/objects/lib/__jest__/process_import_response.test.ts

```diff
@@ -36,32 +41,28 @@ describe('processImportResponse()', () => {
       successCount: 0,
       errors: [
         {
-          obj: {
-            type: 'a',
-            id: '1',
-          },
+          type: 'a',
+          id: '1',
```

Comment:
Test dataset did not match real structure. Adapted test data and updated snapshots.